scripts/actions/103sucknipples.gd

```diff
@@ -0,0 +1,464 @@
+	return"[name2] squeeze[s/2] [his2] eyes shut and {^moan[s/2] involuntarily:can't help but moan:gasp[s/2] at the intense sensation:tremble[s/2] at the intense sensation}."
+
+
+func getDomReaction(member):
+	var actions
+	var text
+
+	if member.sens < 300:
+		if givers.size() > 1:
+			text = "both of you"
+		elif givers[0].person.sex == "male":
+			text = "boy"
+		else:
+			text = "girl"
+		actions = [
+			"[name2] {^purr[s/2]:moan[s/2]} {^softly:happily:encouragingly} at the sensation.",
+			"[name2] {^purr[s/2]:moan[s/2]} {^softly:happily:encouragingly} at the sensation, {^whispering:muttering:saying}: [color=yellow]{^Yes! Do that some more!:Go on, " + text + "! Don't stop!}[/color]",
+			"[name2] {^purr[s/2]:moan[s/2]} {^softly:happily:encouragingly} at the sensation, {^whispering:muttering:saying}: [color=yellow]{^That works. Do continue!:Yes, that's good. Keep going, " + text + "!}[/color]"
+		]
+	elif member.sens < 800:
+		if givers.size() > 1:
+			text = "Keep going, both of you!"
+		elif givers[0].person.sex == "male":
+			text = "Good boy!"
+		else:
+			text = "Good girl!"
+		actions = [
+			"[name2] {^moan[s/2]:purr[s/2]:sigh[s/2]} in {^pleasure:arousal:ecstasy}.",
+			"[name2] {^moan[s/2]:purr[s/2]:sigh[s/2]} in {^pleasure:arousal:ecstasy}, {^whispering:muttering:crying}: [color=yellow]{^Yesss!:" + text + ":Ooh, yes! " + text + ":I like that!}[/color]",
+			"[name2] {^moan[s/2]:purr[s/2]:sigh[s/2]} in {^pleasure:arousal:ecstasy}, {^whispering:muttering:crying}: [color=yellow]{^Ooh, I like that!:That's lovely!:Don't stop! " + text + "}[/color]"
+		]
+	else:
+		actions = [
+			"[name2] {^tremble[s/2]:quiver[s/2]:shiver[s/2]} {^at the slightest touch:in response:ecstatically} {^as [he2] rapidly near[s/2] orgasm:as [he2] approach[es/2] orgasm:as [he2] edge[s/2] toward orgasm:}."
+		]
+	return globals.randomitemfromarray(actions)
+
+
+# Return true if the given member was embarrassed by her chest size. Grown females with low
+# confidence and small tits can feel embarrassed, and suffer a penalty.
+func isEmbarrassed(member):
+	var isFemale = member.person.sex == "female"
+	var isTeenOrOlder = member.person.age != "child"
+	var hasSmallTits = member.person.titssize in ['masculine','flat','small']
+	var hasLowConfidence = member.person.conf <= 40
+	var failedConfidenceCheck = rand_range(-10, 45) > member.person.conf
+
+	return isFemale && isTeenOrOlder && hasSmallTits && hasLowConfidence && failedConfidenceCheck
+
+
+func getNormalReaction(member):
+	var actions
+
+	if member.sens < 300:
+		if isEmbarrassed(member):
+			actions = [
+				"[name2] {^gasp[s/2] and blush[es/2]:moan[s/2] and flush[es/2]:cringe[s/2]}{^ as [his2] nipples are stimulated: as [his2] nipples are teased}, {^whispering:and softly says}: [color=yellow]{^They're a bit small, aren't they?:Are they too small, you think?:Sorry! I wish they were bigger.}[/color]"
```

Comment:
Hmm! On second thought, this will sound like she's referring to the size of her nipples, not her breasts. I'll update it to make it clear.